consensus-peer/connection/grpc-logger.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package connection
+
+import (
+	"github.com/op/go-logging"
+	"google.golang.org/grpc/grpclog"
+)
+
+type lowVerbosityGRPCLog struct {
+	*logging.Logger
+}
+
+func (l *lowVerbosityGRPCLog) Fatalln(args ...interface{}) {
+	l.Fatal(args...)
+}
+
+func (l *lowVerbosityGRPCLog) Print(args ...interface{}) {
+	l.Debug(args...)
+}
+
+func (l *lowVerbosityGRPCLog) Printf(format string, args ...interface{}) {
+	l.Debugf(format, args...)
+}
+
+func (l *lowVerbosityGRPCLog) Println(args ...interface{}) {
+	l.Debug(args...)
+}
+
+func init() {
+	l := &lowVerbosityGRPCLog{logging.MustGetLogger("grpc")}
+	logging.SetLevel(logging.INFO, "grpc")
```

Comment:
this way we can optionally switch to DEBUG to observe grpc output, without having to change this code.